e2e/preview/components/test_transcriber.py

```diff
@@ -0,0 +1,39 @@
+import os
+from pathlib import Path
+
+import pytest
+
+from haystack.preview.components.audio.whisper_remote import RemoteWhisperTranscriber
+
+
+SAMPLES_PATH = Path(__file__).parent.parent / "test_files"
```

Comment:
We have a [fixture for this](https://github.com/deepset-ai/haystack/blob/9d52998b2546a135a469d6659c687f2e393a7c05/e2e/conftest.py#L24-L26), might as well use it. 👀

Reply:
I was considering making a separate folder for the files used in the preview tests, so ease the transition later on. I'll make a separate fixture.